test/softmax/test_softmax_util.hpp

```diff
@@ -0,0 +1,114 @@
+#include <vector>
+#include <iostream>
+#include <half.hpp>
```

Comment:
I think `half.hpp` is not needed

Reply:
Changed in https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/284/commits/9bea75f6d9efe3d2af63739050a16a7df68d6efe